addons/account/models/partner.py

```diff
@@ -234,15 +234,15 @@ class ResPartner(models.Model):
     def _credit_debit_get(self):
         tables, where_clause, where_params = self.env['account.move.line']._query_get()
         where_params = [tuple(self.ids)] + where_params
-        self._cr.execute("""SELECT l.partner_id, act.type, SUM(l.amount_residual)
-                      FROM account_move_line l
-                      LEFT JOIN account_account a ON (l.account_id=a.id)
+        self._cr.execute("""SELECT account_move_line.partner_id, act.type, SUM(account_move_line.amount_residual)
+                      FROM account_move_line
+                      LEFT JOIN account_account a ON (account_move_line.account_id=a.id)
                       LEFT JOIN account_account_type act ON (a.user_type_id=act.id)
                       WHERE act.type IN ('receivable','payable')
-                      AND l.partner_id IN %s
-                      AND l.reconciled IS FALSE
-                      """ + where_clause + """
-                      GROUP BY l.partner_id, act.type
+                      AND account_move_line.partner_id IN %s
+                      AND account_move_line.reconciled IS FALSE
+                      AND """ + where_clause + """
```

Comment:
¿Is posible add the AND without change this:
https://github.com/OCA/ocb/blob/6a366b216b3d119cd5546c877a2f77b5342cb452/openerp/osv/query.py#L30-L31?

If _where_clause_ is not established this error is showed:
openerp.sql_db: Programming error: syntax error at or near "GROUP"
LINE 9:                       GROUP BY account_move_line.partner_id,...
                             ^
, in query SELECT account_move_line.partner_id, act.type, SUM(account_move_line.amount_residual)
                     FROM account_move_line
                     LEFT JOIN account_account a ON (account_move_line.account_id=a.id)
                     LEFT JOIN account_account_type act ON (a.user_type_id=act.id)
                     WHERE act.type IN ('receivable','payable')
                     AND account_move_line.partner_id IN %s
                     AND account_move_line.reconciled IS FALSE
                     AND 
                     GROUP BY account_move_line.partner_id, act.type